### ISSUE TITLE
PTL uses deprecated TBB headers

### ISSUE THREAD

**Issue by amadio** [jrmadsen/PTL#4]
While compiling Geant4 10.7 beta 01, many warnings show up from usage of deprecated headers from TBB:
```
In file included from geant4.10.07.b01/source/externals/ptl/include/PTL/ThreadData.hh:35,
                 from geant4.10.07.b01/source/externals/ptl/include/PTL/ThreadPool.hh:34,
                 from geant4.10.07.b01/source/externals/ptl/include/PTL/TaskGroup.hh:35,
                 from geant4.10.07.b01/source/externals/ptl/include/PTL/TBBTaskGroup.hh:32,
                 from geant4.10.07.b01/source/tasking/include/G4TBBTaskGroup.hh:42,
                 from geant4.10.07.b01/source/tasking/include/G4TaskRunManager.hh:43,
                 from geant4.10.07.b01/source/tasking/include/G4RunManagerFactory.hh:40,
                 from geant4.10.07.b01/source/tasking/src/G4RunManagerFactory.cc:28:
/usr/include/tbb/task_scheduler_init.h:21:154: note: #pragma message: TBB Warning: tbb/task_scheduler_init.h is deprecated. For details, please see Deprecated Features appendix in the TBB reference manual.
 #pragma message("TBB Warning: tbb/task_scheduler_init.h is deprecated. For details, please see Deprecated Features appendix in the TBB reference manual.")
```

**Comment by jrmadsen** [comment]
Is this from an external PTL build or the version in Geant4?

**Comment by jrmadsen** [comment]
Oh ha nevermind. Clearly the internal version

**Comment by amadio** [comment]
Yes, the internal version. If it has been fixed in the upstream version, probably just updating it in Geant4 should be enough.

**Comment by jrmadsen** [comment]
Closed via #5
